src/EventExport/FileWriter/TabularDataFileWriter.php

```diff
@@ -38,29 +51,9 @@ protected function writeHeader()
      */
     public function exportEvent($event)
     {
-        $event = json_decode($event);
-        $row = $this->emptyRow();
+        $eventRow = $this->eventFormatter->formatEvent($event);
 
-        foreach ($this->includedProperties as $property) {
-            $column = $this->columns()[$property];
-
-            if (isset($event->{$column['property']}) || isset($event->{'@' . $column['property']})) {
```

Comment:
@cyberwolf I moved this check down to the column callback because they are no longer mapped directly to a property.